Makefile

```diff
@@ -2,7 +2,10 @@ init:
 	poetry install
 
 test:
-	poetry run coverage run -m pytest && poetry run coverage report -m
+	poetry run coverage run -m pytest tests/unit/ && poetry run coverage report -m
```

Comment:
Tests have been separated into unit and integration folders so that they can be run separately